src/com/google/enterprise/adaptor/prebuilt/RegexDecisionFilter.java

```diff
@@ -255,18 +262,24 @@ public static SkipDocumentFilter create(Map<String, String> cfg) {
       log.config("pattern set to " + patternString);
     }
 
-    String skipOnMatchString = cfg.get("skipOnMatch");
-    if (skipOnMatchString != null) {
-      skipOnMatchString = skipOnMatchString.trim();
-    }
-    if (!Strings.isNullOrEmpty(skipOnMatchString)) {
-      skipOnMatch = Boolean.parseBoolean(skipOnMatchString);
+    String decideOnMatchString = getTrimmedValue(cfg, "decideOnMatch");
```

Comment:
PJ wants a better name here. What about "when" or "result", with values of "found" and "not-found" (which is actually what the code does; it uses find() rather than matches()):

pattern = 666
when = found
decision = do-not-index